crates/configuration/testing/snapshots/0001-big-network.toml

```diff
@@ -63,6 +64,7 @@ balance = 1000000000
 id = 2000
 chain = "myotherparachain"
 add_to_genesis = true
+onboard_as_parachain = true
```

Comment:
since default is `true` this should not be needed

Reply:
> since default is `true` this should not be needed

That true, but I think tracking if the user `set`/`not set` the value to add or not the field to the `toml` is add more complexity, and since this fields will be calculated later in the `ComputeChainSpec` having this value in the `toml` is ok. I mean, if we `skip` the field if the value is `true` we will need to add an extra state tracking to check if the `true` is the default value or the value set by the user, since if the the user `set` the value we should show it in the config. TLDR, I think is ok leave the field.

Reply:
@wirednkod @pepoviola thanks for the feedback, that's true that we could skip it using a custom function and the following serde macro attribute like this: ```#[skip_if(is_true)]```

wdyt ?

Reply:
I think it is valid - allowing less typing and config where not needed. since it will be clearly mentioned in documentation, I think this could work

Reply:
@l0r1s / @wirednkod, oks we can skip it for now and check the feedback in the future. Also, just to add some context, this config `dumping` is a helper that we want to use as integration and is not the main path for endusers of the sdk.
Thanks!